### ISSUE TITLE
Accessibility regression: Tooltips don't persist on hover any longer

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#23958]
**Describe the bug**
Tooltips should persist on hover.

This feature was previously implemented in https://github.com/WordPress/gutenberg/issues/7004 / https://github.com/WordPress/gutenberg/pull/8033

However, in the Gutenberg version currently shipped in WordPress 5.5 Beta 2, tooltips don't persist on hover any longer.

I made two animated GIF to better illustrate:

WordPress 5.4:

![tooltips persist](https://user-images.githubusercontent.com/1682452/87545776-d21ad280-c6a8-11ea-8dca-1d879cb709fd.gif)

WordPress 5.5 Beta 2:

![tooltips do not persist](https://user-images.githubusercontent.com/1682452/87545803-d941e080-c6a8-11ea-862b-152011da3b1c.gif)

There are 3 requirements for accessible tooltips. For more details, see https://github.com/WordPress/gutenberg/pull/8033

Reference:

**WCAG Success Criterion 1.4.13 Content on Hover or Focus**
https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

- dismissable
- hoverable
- persistent

Worth noting that #8033 made the tooltips persist but the "dismissable" part is missing. This part is still waiting for a solution,  see https://github.com/WordPress/gutenberg/issues/15145

In the meantime, the lack of persistence on hover is a clear regression "in trunk" and should be remediated before WordPress 5.5 is released.

**Comment by ntsekouras** [comment]
Τhanks for reporting this @afercia!
